playbook/app/pb_kits/playbook/pb_selectable_card/docs/_selectable_card_error.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import { Body, SelectableCard, Title } from '../..'
+
+const SelectableCardError = (props) => {
+  const [football, setFootball] = useState(false)
+  const [basketball, setBasketball] = useState(false)
+  const [baseball, setBaseball] = useState(false)
+
+  return (
+    <div>
+      <Title
+          dark
```

Comment:
You'll need to replace `dark` with `{...props}` here or the Title kit will be in dark mode even if the global toggle is set to light mode.

Reply:
++